packaging/nnstreamer.spec

```diff
@@ -184,6 +185,10 @@ BuildRequires:  pkgconfig(ovxlib)
 BuildRequires:  pkgconfig(amlogic-vsi-npu-sdk)
 %endif
 
+%if 0%{?grpc_support}
+BuildRequires:  grpc-devel
```

Comment:
CHeck if grpc is available in 1. F-Hub/DA and 2. VD.
If not available, add GBM conditions (Refer to LINE 20, 62, 68)

Reply:
Oh, I think its version might be not matched. I will disable grpc in both profiles. Thanks.